R/tsplot.R

```diff
@@ -142,7 +142,7 @@ tsplot.list <- function(tsl,
   } else{
     if(left_as_bar) return("Finding ticks automatically when stacking values is not supported yet. Please use manual_value_ticks.")
     
-    left_ticks <- do.call(find_ticks_function, list(tsl_r, theme$y_grid_count))
+    left_ticks <- do.call(find_ticks_function, list(tsl_r, theme$y_grid_count, theme$preferred_y_gap_sizes))
```

Comment:
Is this in absolute terms? If so, it's likely to unflexible. there might be millions or billions or trillions or ... involved.